runtime/compiler/CMakeLists.txt

```diff
@@ -225,6 +225,8 @@ set(J9_INCLUDES
 # Makefiles
 set(J9_SHAREDFLAGS
 	${OMR_PLATFORM_COMPILE_OPTIONS}
+)
+set(J9_CFLAGS
```

Comment:
This will break jitserver by ignoring the value set at [line 87](https://github.com/eclipse/openj9/blob/master/runtime/compiler/CMakeLists.txt#L87).

Reply:
I'm tempted to deal with this in a different PR. The existing code there is already pretty broken (we already overwrite the CXX_FLAGS they set there). Then we search for the openssl library, but then never use it? It will take me some time to untangle this, and figure out what what is supposed to be happening here

Reply:
It seems `OPENSSL_CFLAGS` will only have a `-I` option which doesn't need to be added to `J9_CFLAGS` or `J9_CXXFLAGS` because `include_directories()` is used at [line 100](https://github.com/eclipse/openj9/blob/master/runtime/compiler/CMakeLists.txt#L100): lines 86-89 can simply be removed.